apache-rat-core/src/main/java/org/apache/rat/config/exclusion/fileProcessors/GitFileProcessor.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.rat.config.exclusion.fileProcessors;
+
+import org.apache.rat.document.impl.DocumentName;
+
+/**
+ * @see <a href='https://git-scm.com/docs/gitignore'>.gitignore documentation</a>
+ */
+public class GitFileProcessor extends DescendingFileProcessor {
```

Comment:
Will the library we have introduced in release 0.16.0 onwards be used to perform the gitignore parsing or can this lib be removed completely by your PR?

Reply:
I mean 
```
    <dependency>
      <groupId>nl.basjes.gitignore</groupId>
      <artifactId>gitignore-reader</artifactId>
      <version>1.6.0</version>
    </dependency>
```

Reply:
Dpeendency removed. RAT still compiles ;)